camera_control/requirements.txt

```diff
@@ -1,3 +1,3 @@
 imutils
 opencv-python
-
+PySide6
```

Comment:
We should use specific versions here to make sure we don't run into issues for using different versions

Reply:
Agreed - does pip let you download any version on any OS? Or do we need to pick a version that is old enough to be available on the oldest OS version that any of us are running? I'm on Ubuntu 24.04 if that matters.

Reply:
I'm on MacOS 14.6.1.  My guess is that if the version is not available for your OS it would just fail.

Reply:
What version of Python are you using?  I'm on 3.12.2

Reply:
BTW, I just rebased master and tried it again, and now even though VSCode is still showing the errors, the app is working.  Before it was crashing when trying to run predictions

Reply:
What crash were you seeing before? The `Format_RGB888` problem, or something else?

Reply:
The crash message was not very clear, but it didn't explicitely mention `FormatRTGB888`. I didn't dig at the time, just fixed the issues that VSCode was showing me.  
I think what could be happening is that I have the Pylance plugin installed, which is the one showing the errors in VSCode.   It may give a false error if they did some hack thing that the static checker can't follow, and if you don't have the plug in (do you?), you wouldn't see the error.